modules/ROOT/pages/_partials/mmp-deploy-to-rtf.adoc

```diff
@@ -299,6 +299,8 @@ Configuration example:
 
 | `generateDefaultPublicUrl` | When this parameter is set to true, Runtime Fabric generates a public URL for the deployed application.
 
+| `disableAmLogForwarding` | Disables application logs forwarding to Anypoint Monitoring. By default, it is set to `false`.
```

Comment:
What's the meaning?

- `disableAmLogForwarding` disables application-log forwarding to Anypoint Monitoring? or,
- `disableAmLogForwarding` disables application logs by forwarding them to Anypoint Monitoring?